### DISCUSSION TITLE
Error creating a database connection. (An error occurred during DNS resolution

### DISCUSSION THREAD

**Discussion by Lagrafabdessalem** [prisma/prisma#15939] [Q&A]
### Bug description

![Screenshot at 2022-10-23 09-34-30](https://user-images.githubusercontent.com/76662746/197382606-18323376-5dd9-4979-aed5-036a77594975.png)
I'm using linux ubuntu and I installed all things correctly, How I solve It ?

### Prisma information

```prisma
generator client {
  provider        = "prisma-client-js"

}

datasource db {
  provider = "mongodb"
  url      = env("MONGODB_URL")
}

model Account {
  id                 String  @id @default(auto()) @map("_id") @db.ObjectId
  userId             String
  type               String
  provider           String
  providerAccountId  String
  refresh_token      String? @db.String
  access_token       String? @db.String
  expires_at         Int?
  token_type         String?
  scope              String?
  id_token           String? @db.String
  session_state      String?
  user User @relation(fields: [userId], references: [id], onDelete: Cascade)
  @@unique([provider, providerAccountId])
}

model Session {
  id           String   @id @default(auto()) @map("_id") @db.ObjectId
  sessionToken String   @unique
  userId       String
  expires      DateTime
  user         User     @relation(fields: [userId], references: [id], onDelete: Cascade)
}

model User {
  id             String    @id @default(auto()) @map("_id") @db.ObjectId
  name           String?
  email          String?   @unique
  emailVerified  DateTime?
  image          String?
  username       String?  @unique
  accounts       Account[]
  sessions       Session[]
  createdAt      DateTime  @default(now())
  updatedAt      DateTime  @updatedAt
}

model VerificationToken {
  id         String  @id @default(auto()) @map("_id") @db.ObjectId
  identifier String
  token      String   @unique
  expires    DateTime
  @@unique([identifier, token])
}
```

```ts
import NextAuth from "next-auth";
import GoogleProvider from "next-auth/providers/google";
import { PrismaAdapter } from "@next-auth/prisma-adapter";
import { PrismaClient } from "@prisma/client";

const prisma = new PrismaClient();

export default NextAuth({
  adapter: PrismaAdapter(prisma),
  providers: [
    GoogleProvider({
      clientId: process.env.GOOGLE_CLIENT_ID as string,
      clientSecret: process.env.GOOGLE_CLIENT_SECRET as string,
    }),
  ],
  secret: process.env.NEXTAUTH_SECRET,
});
 
```

### Environment & setup

- OS: Debian

### Prisma Version
4.5.0
```

```

**Comment by austincrim** [comment]
Hey @Lagrafabdessalem 👋,

Could you provide a minimal reproduction repository? There are a lot of potential issues here, a minimal repo is the best way to proceed.

**Reply by pantharshit00** [reply]
👋 Hello @Lagrafabdessalem,  did you perhaps have a chance to check @austincrim's recommendation? Let us know if you still have any questions!
 
 Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Reply by austincrim** [reply]
Hey @pantharshit00, feel free not to `@` tag me in these responses. I don't need a ping on these threads 🙂.

**Reply by pantharshit00** [reply]
No problem! I will keep that in mind.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
